server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/ModelServerEndpointException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019 Amazon.com, Inc. or its affiliates.
```

Comment:
The license header seems inconsistent in the files. At places it is 2018, at others it is 2019.